packages/sit-onyx/src/components/OnyxTable/types.ts

```diff
@@ -9,4 +9,11 @@ export type OnyxTableProps = DensityProp & {
    * If `true`, vertical borders will be displayed in addition to the default horizontal borders.
    */
   withVerticalBorders?: boolean;
+  /**
+   * Whether the table will scroll internally when a CSS max height/width is exceeded.
+   * - with scrolling internally, the table header will stick to the top of the **table**
+   * - without scrolling internally, the table header will stick to the top of the **page**
+   * - Never set a max-height/width on the table without setting `withInternalScrolling` to true.
+   */
+  withInternalScrolling?: boolean;
```

Comment:
I don't get the purpose of this prop to be honest^^ If the table height is limited via CSS, it should always be scrollable

Reply:
we could flip the default - let's discuss it tomorrow

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/5668074e-b9a3-47c5-9944-53e7f5e1c77b)
 done :D